src/components/Review.jsx

```diff
@@ -0,0 +1,46 @@
+import { useParams } from "react-router-dom";
+import { fetchReviews, fetchReviewComments } from "./api";
+import { useEffect, useState } from "react";
+
+const Review = () => {
+	const { review_id } = useParams();
+	const [review, setReview] = useState({});
+	const [comments, setComments] = useState([]);
```

Comment:
We aren't worried about comments at this stage, as that's the next ticket. This can be removed from this PR and added to the branch for the next ticket.